### PR TITLE
Swagger

### PR DESCRIPTION
**Pull request by jumasheff**

### CODE REVIEW

File: backend/configs/settings.py [hunk 1]
```diff
@@ -170,4 +171,6 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/2.2/howto/static-files/
 
-STATIC_URL = '/static/'
\ No newline at end of file
+STATIC_URL = '/static/'
+
+REST_FRAMEWORK = {'DEFAULT_SCHEMA_CLASS': 'rest_framework.schemas.coreapi.AutoSchema' }
```
Comment (jumasheff) [inline backend/configs/settings.py]: With this, you are rewriting the same variable on line 84.

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,7 @@
 Django==2.2
 django-authtools==1.7.0
 djangorestframework==3.12.2
-djangorestframework-simplejwt==4.6.0
+djangorestframework-simplejwt==4.4.0
```
Comment (jumasheff) [inline requirements.txt:4]: Why did you change the version?

File: requirements.txt [hunk 3]
```diff
@@ -1,6 +1,7 @@
...
 dj-rest-auth==2.1.1
 django-allauth==0.42.0
+django-rest-swagger
```
Comment (jumasheff) [inline requirements.txt]: What is the version number of this package?

File: backend/configs/settings.py (hunk 1 above)
Comment (ualmaz) [reply backend/configs/settings.py]: Another mystery. I was getting the following error without this variable. 
AttributeError: 'AutoSchema' object has no attribute 'get_link'

I checked it today and it's working without any error.   
swagg

I have commented out this variable.

File: requirements.txt (hunk 2 above)
Comment (ualmaz) [reply requirements.txt:4]: djangorestframework-simplejwt==4.6.0 version is not available anymore. 
error
So I have installed version 4.4.0. Maybe that's a temporary issue. I will check again later if version 4.6.0 is critical for the project.

File: requirements.txt (hunk 3 above)
Comment (ualmaz) [reply requirements.txt]: django-rest-swagger==2.2.0
requirement.txt is updated with other newly installed software.

Comment (jumasheff) [reply requirements.txt]: Can't see the updates :) Make sure you've pushed them.

File: backend/configs/settings.py (hunk 1 above)
Comment (jumasheff) [reply backend/configs/settings.py]: `REST_FRAMEWORK = {'DEFAULT_SCHEMA_CLASS': 'rest_framework.schemas.coreapi.AutoSchema' }`
On line 94 we already have this variable. Instead of commenting, you should add its key/value pair to it (line # 94).

File: requirements.txt (hunk 2 above)
Comment (jumasheff) [reply requirements.txt:4]: I can see that you have added a lot of things coming with the packages we have in the original requirements file.
Please, let's reduce the list to the original one (and just add the swagger package).
Thank you!

File: backend/configs/settings.py (hunk 1 above)
Comment (ualmaz) [reply backend/configs/settings.py]: Fixed.

File: requirements.txt (hunk 2 above)
Comment (ualmaz) [reply requirements.txt:4]: Done!

File: requirements.txt [hunk 4]
```diff
@@ -1,6 +1,9 @@
 Django==2.2
 django-authtools==1.7.0
 djangorestframework==3.12.2
-djangorestframework-simplejwt==4.6.0
+djangorestframework-simplejwt==4.4.0
 dj-rest-auth==2.1.1
 django-allauth==0.42.0
+django-rest-swagger==2.2.0
+
+
```
Comment (jumasheff) [inline requirements.txt]: Nitpick: remove empty lines :)

File: requirements.txt [hunk 5]
```diff
@@ -1,6 +1,9 @@
 Django==2.2
 django-authtools==1.7.0
 djangorestframework==3.12.2
-djangorestframework-simplejwt==4.6.0
+djangorestframework-simplejwt==4.4.0
```
Comment (jumasheff) [inline requirements.txt:4]: Let's test things with `djangorestframework-simplejwt==4.6.0`.
We should always try to use the newest versions.

**Review by jumasheff — COMMENTED** [review]
Can you please take a loom video of the swagger being really used? As in, log in and show all the endpoints and stuff.

File: requirements.txt (hunk 4 above)
Comment (ualmaz) [reply requirements.txt]: :D

File: requirements.txt (hunk 5 above)
Comment (ualmaz) [reply requirements.txt:4]: swagger_in_use

**Review by ualmaz — APPROVED** [review]
